### PR TITLE
Cleaned-up imports in test files related to datasets

### PR DESCRIPTION
**Pull request by prabhat00155**

### CODE REVIEW

**Review by NicolasHug — APPROVED** [review]
Thanks @prabhat00155 ,

In general I'm not too keen on re-ordering imports, as it breaks git blame for little benefits. Maybe others have different opinions (@fmassa @datumbox)?

I'll approve anyway as the removal of `HAS_PYAV` and others unused imports is good!

(Also: merge conflicts ;)  )

**Comment by datumbox** [comment]
No strong opinions from my side on the order of the imports. As a general "policy", as Nicolas said, we do try to avoid rewriting git-blame when possible. Having said that, I'm guilty of rewriting the history of large parts to refactor, so I can't cast the first stone. 😄

**Comment by prabhat00155** [comment]
Makes sense. I was trying to clean up unused imports and while I was at it, make it pep8 compliant as well.

File: test/test_datasets.py [hunk 1]
```diff
@@ -1,44 +1,31 @@
...
+import numpy as np
+import torch
+import torch.nn.functional as F
+from torchvision import datasets
 
 try:
     import scipy
     HAS_SCIPY = True
 except ImportError:
     HAS_SCIPY = False
```
Comment (pmeier) [inline test/test_datasets.py:27]: This is also no longer needed. @prabhat00155 could you send a follow-up PR?

**Pull request merged**